README.md

```diff
@@ -2,18 +2,57 @@
 
 > **Note**: JFrog Workers is the recommended cloud-native solution for extending the JFrog Platform (including Artifactory). While user plugins are still supported, we recommend using JFrog Workers where possible for better scalability, security, and performance. [Learn more about JFrog Workers](https://jfrog.com/help/r/ybbUNZGwwAmzW2qGyL9Zdw/I4E5sOhWWpIHHfdV37__Iw).
 
-## Upgrade Notice
+## Upgrade Notice: Groovy 4 Compatibility
 
-1. **Groovy 4 Upgrade**: Artifactory is upgraded to Groovy 4 in version *7.101.0* (released on 25th Nov 2024 for cloud), which leads user plugins compatible with groovy 4. The promotion plugin has been updated for compatibility with Groovy 4. This upgrade may affect existing user plugins, so ensure you test your plugins after upgrading.
+**Artifactory Version: 7.101 (released November 25, 2024 for cloud)**
 
-   - **Branch Compatibility**:
-     - **master**: Artifactory 7.101.0 & above
-        > **Note**: Groovy 3 no longer supported on master branch
-     - **artifactory-groovy-3**: Artifactory 7.100.0 & below
+Artifactory has been upgraded to Groovy 4 starting from version 7.101. This includes several important updates, most notably the promotion plugin (JFrog Supported), updated for compatibility with Groovy 4. However, this upgrade may break compatibility with your custom plugins developed for earlier Groovy versions (Groovy 3 or older). To learn the differences between Groovy 3 and 4, refer to [Release notes for Groovy 4.0
+](https://groovy-lang.org/releasenotes/groovy-4.0.html)
 
-1. **JDK17 Compatibility**: With the introduction of JDK17 support in Artifactory version **7.43** and above, many older user plugins may no longer be compatible. Deprecated plugins are now moved to the [deprecated directory](http://github.com/jfrog/artifactory-user-plugins/tree/master/deprecated-plugins).
+**JFrog Supported Plugins**
 
-Please consider migrating to JFrog Workers for cloud-native and future-proof solutions.
+- **Promotion Plugin**
+  
+    The promotion plugin has been updated to work with Groovy 4. If you're using this plugin in your environment, redeploy it after upgrading to Groovy 4.
```

Comment:
last line should say upgrading of Artifactory cloud version 7.102 instead of groovy 4. As groovy 4 is part of RT release

Reply:
@nagarajuv-jfrog we also need to mentioned about deprecated plugin. We have not tested them, if customer is using they need to test them & change accordingly.

Reply:
> 7.102

it's 7.101. right?

Reply:
yes